cmd/bgtasks/gotimestatus.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-redis/redis"
+	"github.com/gobridge/gopherbot/internal/poller/gotimestatus"
+	"github.com/rs/zerolog"
+	"github.com/slack-go/slack"
+)
+
+func goTimeStatusNotifyFactory(logger zerolog.Logger, c *slack.Client, channelID string, shadowMode bool) gotimestatus.NotifyFunc {
+	return func(ctx context.Context, statusURL string) error {
+		if shadowMode {
+			logger.Info().
+				Bool("shadow_mode", true).
+				Msgf("would send GoTime Social status %s", statusURL)
+
+			return nil
+		}
+
+		// urls must be enclosed in `<>`. See: https://api.slack.com/reference/messaging/link-unfurling
+		text := fmt.Sprintf("<%s>", statusURL)
+		opts := []slack.MsgOption{
+			slack.MsgOptionUsername("Changelog"),
+			slack.MsgOptionIconURL("https://cdn.changelog.social/accounts/avatars/109/365/688/871/983/824/original/5d1bcf4960706353.png"),
```

Comment:
This controls how the username / icon will appear in chat. I chose the changelog logo and "changelog" as the name on this first pass.